print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
```

Comment:
Assuming this is for a linked list, it may help to add validations to ensure next is also a Node -- Otherwise, some comments about what each field represents could help clarify the constructor's post conditions.

Reply:
Also, can next node be be given default null value to account for if the next value is not given (end of list)?

Reply:
Don't know if this is possible in js, maybe will be taken care of in the linked list class